fuel-core-interfaces/src/relayer.rs

```diff
@@ -106,10 +106,10 @@ pub trait RelayerDb:
 
     /// Apply validators diff to validator set and update validators_da_height. This operation needs
     /// to be atomic.
-    async fn apply_validator_diffs(&mut self, da_height: DaBlockHeight, changes: &HashMap<Address,(ValidatorStake,Option<Address>)>) {
+    async fn apply_validator_diffs(&mut self, da_height: DaBlockHeight, changes: &HashMap<ValidatorId,(ValidatorStake,Option<ConsensusId>)>) {
```

Comment:
Another unrelated nit; we should either take the map as owned so we won't copy every element, or take an iterator.

Reply:
Storage `insert` works on references, so it is internally copied. Either `impl Iter` would look nicer here